src/schemas/profile.json

```diff
@@ -51,6 +51,24 @@
           "title": "farcaster",
           "pattern": "^[a-z0-9-]*$",
           "maxLength": 17
+        },
+        "emailSubscription": {
+          "type": "object",
+          "properties": {
+            "email": {
+              "type": "string",
+              "title": "email",
+              "maxLength": 256
+            },
+            "subscriptions": {
+              "type": "array",
+              "title": "subscriptions",
+              "uniqueItems": true,
+              "items": { "type": "string" }
+            }
+          },
+          "required": [],
```

Comment:
Users can pass subscriptions without email? maybe we should make it required

Reply:
Email will not be available, since not returned by the hub.